app/models/workarea/checkout/steps/shipping.decorator

```
@@ -0,0 +1,8 @@
+module Workarea
+  decorate Checkout::Steps::Shipping, with: :paypal do
+    def update_shipping_service(params)
+      super
+    rescue Checkout::Steps::Shipping::NoAvailableShippingOption
```

Why do we want to swallow this error here?

---

So.....